facebook/src/main/java/com/facebook/GraphRequest.java

```diff
@@ -1706,7 +1706,8 @@ final static void serializeToUrlConnection(
 
         OutputStream outputStream = null;
         try {
-            outputStream = new BufferedOutputStream(connection.getOutputStream());
+            outputStream = connection.getOutputStream();
```

Comment:
This change doesn't seem to reflect the description, and shouldn't be necessary, correct? Only moving the inputStream initialization is necessary?